datasets/fdic/pipelines/_images/run_csv_transform_kub/csv_transform.py

```diff
@@ -0,0 +1,240 @@
+    df = pd.read_csv(str(source_file))
+
+    logging.info(f"Transforming {source_file}... ")
+
+    logging.info("Renaming Columns...")
+    rename_headers(df, rename_mappings)
+
+    if pipeline_name == "locations":
+        logging.info("Replacing bool values...")
+        replace_bool_list = [
+            "main_office",
+            "cbsa_division_flag",
+            "cbsa_metro_flag",
+            "cbsa_micro_flag",
+            "csa_flag",
+        ]
+        replace_bool(replace_bool_list, df)
+        df["cbsa_division_fips_code"] = df["cbsa_division_fips_code"].astype(
+            "Int64", errors="ignore"
+        )
+        logging.info("Replacing date values...")
+        format_date_list = ["date_established", "last_updated"]
+        format_date(format_date_list, df)
+        logging.info("Replacing with null values...")
+        df["cbsa_metro_fips_code"] = df["cbsa_metro_fips_code"].replace(0, "NULL")
+    else:
+        logging.info("Replacing bool values...")
+        replace_bool_list = [
+            "active",
+            "conservatorship",
+            "denovo_institute",
+            "federal_charter",
+            "iba",
+            "inactive_flag",
+            "credit_card_institution",
+            "bank_insurance_fund_member",
+            "insured_commercial_bank",
+            "deposit_insurance_fund_member",
+            "fdic_insured",
+            "saif_insured",
+            "insured_savings_institute",
+            "oakar_institute",
+            "state_chartered",
+            "sasser_institute",
+            "law_sasser",
+            "cfpb_supervisory_flag",
+            "ffiec_call_report_filer",
+            "holding_company_flag",
+            "ag_lending_flag",
+            "ownership_type",
+            "csa_indicator",
+            "cbsa_metro_flag",
+            "cbsa_micro_flag",
+            "cbsa_division_flag",
+            "subchap_s_indicator",
+        ]
+        replace_bool(replace_bool_list, df)
+        logging.info("Replacing date values...")
+        replace_date_list = [
+            "last_updated",
+            "effective_date",
+            "end_effective_date",
+            "deposit_insurance_date",
+            "last_structural_change",
+            "report_date",
+            "reporting_period_end_date",
+            "run_date",
+            "cfpb_supervisory_start_date",
+            "cfpb_supervisory_end_date",
+        ]
+        replace_date(replace_date_list, df)
+        logging.info("Formatting date values...")
+        format_date_list = [
+            "established_date",
+            "last_updated",
+            "effective_date",
+            "end_effective_date",
+            "deposit_insurance_date",
+            "last_structural_change",
+            "report_date",
+            "reporting_period_end_date",
+            "run_date",
+        ]
+        format_date(format_date_list, df)
+        logging.info("Filling null values...")
+        null_list = [
```

Comment:
Please move to pipeline.yaml and incorporate as a variable

Reply:
done